public/index.html

```diff
@@ -37,6 +37,7 @@
         <script src="https://unpkg.com/geotiff@0.4.1/dist/geotiff.browserify.min.js"></script>
         <script src="https://unpkg.com/leaflet-providers@1.1.17/leaflet-providers.js"></script>
         <script src="https://ihcantabria.github.io/Leaflet.CanvasLayer.Field/dist/leaflet.canvaslayer.field.js"></script>
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/leaflet.draw/0.4.12/leaflet.draw.js"></script>
```

Comment:
I just created an issue for it and it won't prevent my approval of the PR, but maybe we should think of being able to have fall backs just in case these CDN's drop our initial request.